squad/ci/backend/lava.py

```diff
@@ -314,10 +318,16 @@ def __parse_results__(self, data, test_job):
                 if handle_lava_suite or result['suite'] != 'lava':
                     suite = result['suite'].split("_", 1)[-1]
                     res_name = "%s/%s" % (suite, result['name'])
+                    res_log = None
+                    if 'log_start_line' in result.keys() and \
```

Comment:
I think you can do `result.get('log_start_line') is not None`, it's less complex

Reply:
IMO `in` is just fine

Reply:
I think it's a good point - 2 lines fewer.